resources/js/components/Signup.tsx

```diff
@@ -0,0 +1,167 @@
+                            <FormLabel htmlFor="lastName" fontSize="sm">
+                                Last Name
+                            </FormLabel>
+                            <Input
+                                id="lastName"
+                                name="lastName"
+                                type="text"
+                                placeholder="Enter your last name"
+                                value={formInputs.lastName}
+                                onChange={onChange}
+                                autoComplete="off"
+                            />
+                        </FormControl>
+                        <FormControl marginBottom="1.5">
+                            <FormLabel htmlFor="email" fontSize="sm">
+                                Email Address
+                            </FormLabel>
+                            <Input
+                                id="email"
+                                name="email"
+                                type="email"
+                                placeholder="Enter your email address"
+                                value={formInputs.email}
+                                onChange={onChange}
+                                autoComplete="off"
+                            />
+                        </FormControl>
+                        <FormControl marginBottom="1.5">
+                            <FormLabel htmlFor="password" fontSize="sm">
+                                Password
+                            </FormLabel>
+                            <Input
+                                id="password"
+                                name="password"
+                                type="password"
+                                placeholder="Enter your password"
+                                value={formInputs.password}
+                                onChange={onChange}
+                            />
+                        </FormControl>
+                        <FormControl marginBottom="3">
+                            <FormLabel htmlFor="confirmPassword" fontSize="sm">
+                                Confirm Password
+                            </FormLabel>
+                            <Input
+                                id="confirmPassword"
+                                name="confirmPassword"
+                                type="password"
+                                placeholder="Confirm your password"
+                                value={formInputs.confirmPassword}
+                                onChange={onChange}
+                            />
+                        </FormControl>
+                        <Button
+                            type="submit"
+                            w="full"
+                            fontWeight="normal"
+                            isLoading={isLoading}
+                            disabled={
+                                formInputs.firstName.trim() === "" ||
```

Comment:
if the inline condition is more than 2 lines, make separate function to handle this

Reply:
Refactored and used a helper function `isDisabled`